src/components/pages/docs.yml

```diff
@@ -7,9 +7,9 @@ newDocument:
 initialAuthor:
   en: Initial author
   de: Erstautor
-filterPlaceholder:
-  en: Filter by title or initial author
-  de: Nach Titel oder Erstautor filtern
+placeholder:
+  en: Search by title or initial author
+  de: Nach Titel oder Erstautor suchen
```

Comment:
Even though we use the input as a filter, we display to the user a search input, so I thought better to phrase this according to the input.

Reply:
Is there a reason why you did not use the new component here as `<FilterInput size="large" ... />`?

Reply:
The only reason is an invalid one actually 😆 I confused the "Documents" page with the "Search" page (not for the first time) and considered that here it should be search.
I shall change it!

Reply:
Changed, thanks!